org.eclipse.scanning.test/src/org/eclipse/scanning/test/points/CompoundTest.java

```diff
@@ -163,7 +163,7 @@ public void testSimpleToDict() throws Exception {
 		PyList gens = (PyList) dict.get("generators");
 		PyDictionary line1 = (PyDictionary) gens.get(0);
 
-		assertEquals("Temperature", (String) ((PyList) line1.get("axes")).get(0));
+		assertEquals("Temperature", ((PyList) line1.get("axes")).get(0));
 		assertEquals("mm", ((PyList) line1.get("units")).get(0));
 		assertEquals(290.0, (double) ((PyList) line1.get("start")).get(0), 1E-10);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "start" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)